core/config/docs/chains-evm.toml

```
@@ -242,6 +242,9 @@ TipCapDefault = '1 wei' # Default
 # (Only applies to EIP-1559 transactions)
 TipCapMin = '1 wei' # Default
 
+# CostMax is the limit gas cost
+CostMax = '1000000000000000000 wei' # 1 ETH in wei # Default
```

When we control the toml input, we can use the simpler units:
```suggestion
CostMax = '1 ether' # Default
```